src/lib.rs

```diff
@@ -174,6 +179,16 @@ pub trait Colorize {
     {
         self.color(Color::TrueColor { r, g, b })
     }
+    fn custom_color(self, color: &CustomColor) -> ColoredString
```

Comment:
Pass `CustomColor` by value. Please document.

Reply:
Can you please elaborate? What do you mean by pass ```CustomColor``` by value?

Reply:
Rather than borrowing `&CustomColor`, the API can leverage the fact that `CustomColor` is `Copy`able (hence the request to derive it), and as such it is more efficient to move by value. So the signature would look like `fn custom_color(self, color: CustomColor) -> ColoredString`.

Avoiding a borrow helps with the ergonomics of the API as well.

Reply:
Ok, it's done!